api/helpers/validateEnvironment.js

```diff
@@ -1,18 +1,15 @@
-const isEmpty = require('underscore').isEmpty;
```

Comment:
I never got a chance to clean this up. So now this is cleaner and a lot of unneeded or unnecessary code is removed.